gcloud-java-core/src/main/java/com/google/gcloud/ExceptionHandler.java

```diff
@@ -48,18 +47,7 @@
   public interface Interceptor extends Serializable {
 
     enum RetryResult {
-
-      RETRY(true), ABORT(false);
-
-      private final boolean booleanValue;
-
-      RetryResult(boolean booleanValue) {
-        this.booleanValue = booleanValue;
-      }
-
-      boolean booleanValue() {
-        return booleanValue;
-      }
+      ABORT, RETRY, PROCEED;
```

Comment:
Can we make the enum values more clear (and the javadoc).
Maybe `NO_RETRY`, `RETRY` and `PROCEED_EVALUATION` (maybe you can think of a better name for the last one).